scripts/drone/pipelines/swagger_gen.star

```diff
@@ -33,7 +33,7 @@ def swagger_gen_step(ver_mode):
             "apk add --update git make",
             "make swagger-clean && make openapi3-gen",
             "for f in public/api-merged.json public/openapi3.json; do git add $f; done",
-            'if [ -z "$(git diff --name-only --cached)" ]; then echo "Everything seems up to date!"; else echo "Please ensure the branch is up-to-date, then regenerate the specification by running make swagger-clean && make openapi3-gen" && return 1; fi',
+            'if [ -z "$(git diff --name-only --cached)" ]; then echo "Everything seems up to date!"; else git diff --cached && echo "Please ensure the branch is up-to-date, then regenerate the specification by running make swagger-clean && make openapi3-gen" && return 1; fi',
```

Comment:
There was another issue later on today where the swagger docs did drift in `main` which stalled most builds. (Basically, a change managed to escape the CI step)

Thread:
https://raintank-corp.slack.com/archives/C04TFFPTSNS/p1709755662500559
PR: https://github.com/grafana/grafana/pull/84037

In the process of fixing that, I brought in this one line of change to print the diff - we needed it to make debugging easier.

I pinged you in the commit that added this: https://github.com/grafana/grafana/pull/84037/commits/2c6037adc432766fffe0deb75b9b377b39b60b0c to hopefully attribute it correctly as it was pilfered directly from this PR.

So, this one tiny piece is already merged. I think all it needs is a .drone.yml regeneration.